packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -319,19 +319,19 @@ curl -v -X PUT \
 
 <ApiOperation method="post" url="/api/v1/inlineHooks/${id}/lifecycle/activate" />
 
-##### Request Parameters
+##### Request parameters
 
 | Parameter  | Description                                                                     | Param Type   | DataType                                    | Required |
 | ---------- | ------------------------------------------------------------------------------- | ------------ | ------------------------------------------- | -------- |
 | id         | The ID of the Inline Hook you want to activate.                                   | Path         | String                                      | TRUE     |
```

Comment:
no period for fragments and insert that

The ID of the Inline Hook that you want to activate